action.yaml

```diff
@@ -61,26 +61,30 @@ runs:
       shell: bash
       env:
         INPUT_VERSION: ${{ inputs.version }}
+        OS_NAME: ${{ runner.os }}
       run: |
-        if [[ "${{ runner.os }}" == "macOS" ]]; then
+        if [ "${INPUT_VERSION}" = "system" ]; then
+          echo "Using system shellcheck: $(command -v shellcheck)"
+          exit 0
+        fi
```

Comment:
Shellcheck is guaranteed to be installed in a GitHub Actions environment. If it for some reason it can't be found, I don't see why we should exit "gracefully". Invoking it will simply fail with "command not found" which should be enough information to the user about what went wrong.